homeassistant/components/hue/config_flow.py

```diff
@@ -246,13 +248,13 @@ async def async_step_init(
                     vol.Optional(
                         CONF_ALLOW_HUE_GROUPS,
                         default=self.config_entry.options.get(
-                            CONF_ALLOW_HUE_GROUPS, False
+                            CONF_ALLOW_HUE_GROUPS, DEFAULT_ALLOW_HUE_GROUPS
                         ),
                     ): bool,
                     vol.Optional(
                         CONF_ALLOW_UNREACHABLE,
                         default=self.config_entry.options.get(
-                            CONF_ALLOW_UNREACHABLE, False
+                            CONF_ALLOW_UNREACHABLE, DEFAULT_ALLOW_UNREACHABLE
```

Comment:
But why is the default set to True. It's introduced in #31713 which clearly shows that it should be False 🤔

Reply:
I have no idea why it's set to true.  We should change the constant to false given that.

Reply:
Yeah. The groups are annoying because they don't have unique IDs :(